scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -406,4 +406,93 @@ describe("import/export end to end", () => {
     const bData = JSON.parse(fs.readFileSync(bPath).toString());
     expect(bData).to.equal(null);
   });
+
+  it("should be able to import/export auth data", async function(this) {
```

Comment:
Please also add a test (or modify this one to also do this) that checks for exporting more than one account, and perhaps a test for no accounts

Reply:
Modified the tests to include two users and added a one for no users. The one with no users has caught real bugs that I've fixed -- thanks a ton!